src/lib/ruby-generator/video.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Define Ruby code generator for Video Blocks
+ * @param {RubyGenerator} Generator The RubyGenerator
+ * @return {RubyGenerator} same as param.
+ */
+export default function (Generator) {
+    Generator.videoSensing_whenMotionGreaterThan = function (block) {
+        block.isStatement = true;
+        const rh = Generator.valueToCode(block, 'REFERENCE', Generator.ORDER_NONE) || 0;
+        return `${Generator.spriteName()}.when(:greater_than, self.video_motion, ${rh}) do\n`;
+    };
+
+    Generator.videoSensing_videoToggle = function (block) {
+        const video_state = Generator.valueToCode(block, 'VIDEO_STATE', Generator.ORDER_NONE);
+        return `turn_video(${video_state})\n`;
```

Comment:
video関連は、なるべく `video_なんたら` というメソッド名にしたいと考えて、リネームしました。

```suggestion
        return `video_turn(${video_state})\n`;
```